PedalCore/Views/Songs/SongDetail/SongDetailView.swift

```diff
@@ -0,0 +1,216 @@
+        VStack(alignment: .leading) {
+            HStack {
+                Text("Pedals")
+                    .font(.footnote)
+                    .foregroundStyle(.secondary)
+                
+                Spacer()
+                
+                Button {
+                        withAnimation {
+                            viewModel.editKnobsPressed()
+                        
+                    }
+                } label: {
+                    Image(systemName: viewModel.isInEditingKnobsMode ? "lock.open.fill" : "lock.fill")
+                        .resizable()
+                        .scaledToFit()
+                        .frame(width: 20, height: 20)
+                        .foregroundStyle(Color.accentColor)
+                        .disabled(viewModel.isInEditingSongMode)
+                }
+            }
+            
+            Divider()
+            
+            VStack {
+                if viewModel.isInEditingKnobsMode {
+                    VStack {
+                        Text("Editing Mode On")
+                            .font(.subheadline)
+                            .fontWeight(.semibold)
+                            .foregroundStyle(Color.accentColor)
+                        
+                        Text("You can adjust the knob level by draggin")
+                            .font(.caption)
+                            .foregroundStyle(Color.accentColor)
+                    }
+                    .padding(.top)
+                }
+                
+                if viewModel.isInEditingSongMode {
+                    Button {
+                        withAnimation {
+                            viewModel.ChangePedalsButtonPressed()
+                        }
+                        
+                    } label: {
+                        Text("Change Pedals")
+                            .fontWeight(.semibold)
+                            .padding()
+                    }
+                }
+                
+                ForEach($viewModel.song.pedals, id: \.signature) { $pedal in
+                    VStack(alignment: .leading) {
+                        Text(pedal.name)
+                            .foregroundStyle(.primary)
+                            .font(.headline)
+                        
+                        Text(pedal.brand)
+                            .font(.subheadline)
+                            .foregroundStyle(.secondary)
+                        
+                        if !viewModel.isInEditingSongMode {
+                            KnobsGridView(knobs: $pedal.knobs, knobStyle: viewModel.isInEditingKnobsMode ? .editing : .reference)
+                                .allowsHitTesting(viewModel.isInEditingKnobsMode)
+                        }
+                    }
+                    .frame(maxWidth: .infinity,
+                           alignment: .leading)
+                    .padding()
+                }
+            }
+            .background(
```

Comment:
```suggestion
            }
            .padding(.bottom, 20)
            .background(
```
I would like to suggest a bottom padding, just to for an aesthetic reason!